src/main/java/com/michelin/ns4kafka/service/executor/TopicAsyncExecutor.java

```diff
@@ -203,6 +203,25 @@ public void deleteTopic(Topic topic) throws InterruptedException, ExecutionExcep
             managedClusterProperties.getName());
     }
 
+    /**
+     * Delete a list of topics.
+     *
+     * @param topics The topics to delete
+     */
+    public void deleteTopics(List<Topic> topics) throws InterruptedException, ExecutionException, TimeoutException {
```

Comment:
Little improvement:
https://github.com/michelin/ns4kafka/blob/ae3e5d9a145a6869c304a56d5dbda0bbb5f4ea37/src/main/java/com/michelin/ns4kafka/service/executor/TopicAsyncExecutor.java#L197 can call `deleteTopics(List<Topics> topics)` like

public void deleteTopic(Topic topic) {
  deleteTopics(List.of(topic));
}

Reply:
Maybe, it is even better to completely remove the method and replace it by `deleteTopics(List<Topic> topics)`.
What do you think?